docs/sources/reference/run.md

```diff
@@ -476,6 +476,7 @@ container:
     --cpuset-cpus="": CPUs in which to allow execution (0-3, 0,1)
     --cpuset-mems="": Memory nodes (MEMs) in which to allow execution (0-3, 0,1). Only effective on NUMA systems.
     --cpu-quota=0: Limit the CPU CFS (Completely Fair Scheduler) quota
+    --oom-kill-disable=false: Do not disable the OOM Killer when OOM happened.
```

Comment:
This should be the same text as the other documents;

`--oom-kill-disable=true|false: Whether to disable OOM Killer for the container or not.`